File: freecodecamp/daily-code-challenge/2026-02-16.js
function getSemifinalMatchups(teams) {
  const standings = teams.map(team => {
    const [name, record] = team.split(": ");
    const [W, OTW, OTL, L] = record.split("-").map(Number);
    
    const points = (W * 3) + (OTW * 2) + (OTL * 1);
    
    return { name, points };
  });

  // Sort by points descending
  standings.sort((a, b) => b.points - a.points);

  return `The semi-final games will be ${standings[0].name} vs ${standings[3].name} and ${standings[1].name} vs ${standings[2].name}.`;
}

File: freecodecamp/daily-code-challenge/2026-02-16.py
def get_semifinal_matchups(teams):
    standings = []
    
    for team in teams:
        name, record = team.split(": ")
        W, OTW, OTL, L = map(int, record.split("-"))
        
        points = (W * 3) + (OTW * 2) + (OTL * 1)
        standings.append((name, points))
    
    # Sort by points descending
    standings.sort(key=lambda x: x[1], reverse=True)
    
    return f"The semi-final games will be {standings[0][0]} vs {standings[3][0]} and {standings[1][0]} vs {standings[2][0]}."
